src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -57,7 +57,10 @@ public function validateActiveUrl($attribute, $value)
 
         if ($url = parse_url($value, PHP_URL_HOST)) {
             try {
-                return count(dns_get_record($url, DNS_A | DNS_AAAA)) > 0;
+                return (bool) @dns_get_record(
```

Comment:
Please don't surpress errors silently here.

Reply:
We had a discussion above and @GrahamCampbell commited this https://github.com/laravel/framework/pull/32369/commits/5ec1ebf556b8d3f16e915096b11251de6b0f9df2

Reply:
Either we should throw an exception when false is returned, or we should suppress the warning. Before this PR we were doing neither.

Reply:
I think he decided to silent it because it was generating noisy warnings on tinker.
![image](https://user-images.githubusercontent.com/56585913/79226668-4c3f7f00-7e4e-11ea-95c5-eecb5d01fb5b.png)